includes/templates/reader-activation-emails/otp.php

```diff
@@ -9,217 +9,278 @@
+
+$post_content =
+	// Main body.
+	'<!-- wp:group {"style":{"spacing":{"padding":{"top":"56px","bottom":"56px","left":"56px","right":"56px"}},"elements":{"link":{"color":{"textColor":"primary","text":"var:preset|color|primary"}}}},"backgroundColor":"secondary","textColor":"secondary-text","className":"has-primary-color has-primary-text-color ' . \Newspack\Emails::POST_TYPE . '-has-secondary-text-color","layout":{"type":"constrained"}} -->
+	<div class="wp-block-group has-primary-color has-primary-text-color has-secondary-text-color has-secondary-background-color has-text-color has-background has-link-color" style="padding-top:56px;padding-right:56px;padding-bottom:56px;padding-left:56px">
 	<!-- wp:site-logo {"width":50} /-->
 	<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
-	<h2 class="wp-block-heading" style="font-style:normal;font-weight:400">' . __( 'Sign in', 'newspack' ) . '</h2>
+	<h2 class="wp-block-heading" style="font-style:normal;font-weight:400">' . __( 'Sign in', 'newspack-plugin' ) . '</h2>
 	<!-- /wp:heading -->
-
 	<!-- wp:paragraph -->
-	<p>' . __( 'Use the following code to login to your account:', 'newspack' ) . '</p>
+	<p>' . __( 'Use the following code to login to your account:', 'newspack-plugin' ) . '</p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:paragraph {"fontSize":"x-large"} -->
 	<p class="has-x-large-font-size"><code>*MAGIC_LINK_OTP*</code></p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
-	<p style="font-style:normal;font-weight:400">' . __( 'You can also log into your account by clicking the following button:', 'newspack' ) . '</p>
+	<p style="font-style:normal;font-weight:400">' . __( 'You can also log into your account by clicking the following button:', 'newspack-plugin' ) . '</p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:buttons {"layout":{"type":"flex","orientation":"horizontal","flexWrap":"nowrap","justifyContent":"left"},"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}},"fontSize":"medium"} -->
-	<div class="wp-block-buttons has-custom-font-size has-medium-font-size" style="font-style:normal;font-weight:400"><!-- wp:button {"textAlign":"center","style":{"color":{"background":"' . esc_attr( $primary_color ) . '"},"border":{"radius":"4px"}}} -->
-	<div class="wp-block-button"><a class="wp-block-button__link has-background has-text-align-center wp-element-button" href="*MAGIC_LINK_URL*" style="border-radius:4px;background-color:' . esc_attr( $primary_color ) . '">' . sprintf( /* Translators: s: site title. */ __( 'Continue to %s', 'newspack' ), '*SITE_TITLE*' ) . '</a></div>
+	<div class="wp-block-buttons has-custom-font-size has-medium-font-size" style="font-style:normal;font-weight:400"><!-- wp:button {"textAlign":"center","backgroundColor":"primary","textColor":"secondary","style":{"border":{"radius":"4px"},"elements":{"link":{"color":{"text":"secondary"}}}}} -->
```

Comment:
Is setting the text color to the secondary color intentional? On the screenshot in PR description it's the contrasting text color, not theme secondary color:

<img width="589" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/7383192/76ac9e3b-dead-435b-a1eb-c64029c4b08c">

Reply:
> Is setting the text color to the secondary color intentional? On the screenshot in PR description it's the contrasting text color, not theme secondary color

Sorry about that. It is. I had another look at the designs (c6T1goPQpkchdh7XBAM6aH-fi-1_10676) and it seems like the button text is meant to be the secondary color.